src/main/java/Trader.java

```diff
@@ -98,6 +98,7 @@ public static void main(String[] args) {
                 new Horse(),
                 new Horse()
                 // TODO: Add Tradable objects here!
+
```

Comment:
Should add object SpaceShuttle. (eg. new SpaceShuttle())

Reply:
Oh oops, forgot to do that